spec/board-spec.rb

```diff
@@ -0,0 +1,70 @@
+require '../lib/game.rb'
```

Comment:
Please, use `require_relative` with the relative path

Reply:
That rule should also be applied in other `*.spec` files

Reply:
Thanks, I just read about the difference in the require, require_relative and require_all methods. This makes sense